### ISSUE TITLE
Button accessibility violations

### ISSUE THREAD

**Issue by serhiitalashko-afk** [highcharts/highcharts#24379]
Buttons with 3 dots (to open menu in columnt header) has accessibility violations described by Storybook.
`<button class="hcg-button" type="button" tabindex="-1">`

Violation description:
```
Element does not have inner text that is visible to screen readers.
aria-label attribute does not exist or is empty.
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty.
Element has no title attribute.
Element does not have an implicit (wrapped) <label>.
Element does not have an explicit <label>.
Element's default semantics were not overridden with role="none" or role="presentation".
```

I believe creating `aria-label` for it should do the trick.

**Comment by stitot** [comment]
Thanks, @serhiitalashko-afk . We'll look into it.
